src/github.com/getlantern/lantern-ui/app/js/app.js

```diff
@@ -256,22 +261,27 @@ var app = angular.module('app', [
     $rootScope.$watch("wsConnected", function(wsConnected) {
       var MILLIS_UNTIL_BACKEND_CONSIDERED_GONE = 10000;
       if (!wsConnected) {
-        // In 11 seconds, check if we're still not connected
-        $timeout(function() {
-          var lastConnectedAt = $rootScope.wsLastConnectedAt;
-          if (lastConnectedAt) {
-            var timeSinceLastConnected = new Date().getTime() - lastConnectedAt.getTime();
-            $log.debug("Time since last connect", timeSinceLastConnected);
-            if (timeSinceLastConnected > MILLIS_UNTIL_BACKEND_CONSIDERED_GONE) {
-              // If it's been more than 10 seconds since we last connect,
-              // treat the backend as gone
-              console.log("Backend is gone");
-              $rootScope.backendIsGone = true;
-            } else {
-              $rootScope.backendIsGone = false;
+        // Temporary workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1192773
+        if (navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
```

Comment:
Yes, it's intentional. This timeout is what shows the message "Lantern is Gone" after 11 seconds in the "waiting for lantern" status.